### ISSUE TITLE
Implement non-bounding-box-affecting data augmentation functions

### ISSUE THREAD

**Issue by mayhemsloth** [mayhemsloth/BTCC-TF-Model#1]
Certain data augmentation techniques for computer vision data do *not* require the need to change the labels to describe the bounding boxes. For example, color operations affect the values of the color channels but do not translate or otherwise transform *where* the object class is within the image itself. Color operations include: color, equalize, contrast, brightness, sharpness, and solarize.

This issue is resolved when data augmentation functionality has been implemented, in use for the Single Class model.

**Comment by mayhemsloth** [comment]
I discovered in the tutorial code where the training augmentation occurs and implemented some color-changing functions so that training data can randomly be changed with these color-changing functions during different epochs. I can do more if I really want to.
